rollup.config.js

```diff
@@ -37,7 +37,7 @@ const transpile = {
   input: 'src/index.tsx',
   external: (source) => {
     // @ethersproject/* modules are provided by ethers
-    return source.startsWith('@ethersproject/')
+    return [...source.startsWith('@ethersproject/'), 'styled-components']
```

Comment:
This change shouldn't be necessary, as this should already be marked external because it is not a dependency.

Reply:
That makes sense, but I'm confused now why the styled-components docs for library maintainers specifically call this out as a necessary step *after* the dev/peer changes. 🤔

Reply:
In our case, this is done through https://www.npmjs.com/package/rollup-plugin-node-externals in the rollup config, similar to the suggested `rollup-plugin-peer-deps-external` solution in the styled-component docs.